src/main/java/com/quizapp/enterprise/controllers/GameController.java

```diff
@@ -55,8 +55,11 @@ public Question nextQuestion(@PathVariable("id") String gameId){
     }
 
     @PostMapping("/checkGuess")
-    public GuessResult checkGuess(@RequestBody Guess userGuess){
-        GuessResult result = gameService.ProcessPlayerGuess(userGuess.getGuess(),userGuess.getGameCode(), userGuess.getQuestionId(), userGuess.getPlayerName());
+    public GuessResult checkGuess(@RequestBody Guess userGuess) throws Exception {
```

Comment:
related to respective handling of null pointer exception change

Reply:
See comments above.